### ISSUE TITLE
How to run this project on colab for own dataset

### ISSUE THREAD

**Issue by rt30** [matterport/Mask_RCNN#2203]
I can't find any file which has all the steps to install the requirements and proceed. I am new to using google colab. Please help in how to setup and run this project on colab

**Comment by rupa1118** [comment]
hi @rt30  to run the project in google colab u need to store all the project files along with the dataset on ur google drive then u need to mount the drive and then u can run on google colab

**Comment by sinanbayraktar** [comment]
You can install the packages it was in the terminal by adding **!** before the statement. You should install them before moving on the rest. And also, you should mount the drive.

**Comment by Shakesbeer333** [comment]
can you please share your Colab notebook?

**Comment by Shakesbeer333** [comment]
!pip install keras==2.0.8
!pip install tensorflow-gpu==1.15.0
!pip install pycocotools
!pip install scikit-image
!pip install imgaug

_you don't need to do "!pip install -r requirements"_

**Comment by Juuustin** [comment]
I am wondering could we open the demo.ipynb in the colab? I use colab to clone the mask rcnn, but there is also ipynb file inside the folder, so how could we open those ipynb files in colab?

**Comment by Krin70** [comment]
> !pip install keras==2.0.8
> !pip install tensorflow-gpu==1.15.0
> !pip install pycocotools
> !pip install scikit-image
> !pip install imgaug
> 
> _you don't need to do "!pip install -r requirements"_

I'm install by above pakages but occured an error:   'Model' object has no attribute 'add_metric' 
till I change keras 2.0.8 to 2.3.0 , colab work!
